container/stream/attach_test.go

```diff
@@ -0,0 +1,102 @@
+package stream
+
+import (
+	"context"
+	"io"
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+// Make sure when there is no I/O on a stream that the goroutines do not get blcoked after the container exits.
```

Comment:
🙈 

```suggestion
// Make sure when there is no I/O on a stream that the goroutines do not get blocked after the container exits.
```

Wondering if there's more to describe for this test; would it make sense to add some links to the GitHub ticket(s)? (mostly to find details in future if needed)

Reply:
@cpuguy83 don't forget this one if you need to push again 😅

Reply:
Fixed